src/prettier.ts

```diff
@@ -0,0 +1,73 @@
+const loadScript = (src: string) => {
+  return new Promise<void>((resolve, reject) => {
+    const scriptElement = document.createElement('script');
+    scriptElement.src = src;
+
+    const timeout = setTimeout(() => {
+      reject('Timeout');
+    }, 15000);
+
+    scriptElement.onload = () => {
+      resolve();
+      clearTimeout(timeout);
+    };
+    document.body.appendChild(scriptElement);
+  });
+};
+
+declare global {
+  export const prettier: typeof import('prettier');
+  export const prettierPlugins: (string | import('prettier').Plugin)[];
+
+  interface Window {
+    define: any;
+  }
+}
+
+const Prettier = () => {
+  let hasLoaded = false;
```

Comment:
Implicit state machine 🙈  (fine for now)

Reply:
😂  I figured this was easier than trying to wrangle `service.send` inside an async function